elements/lisk-p2p/src/peer/base.ts

```diff
@@ -157,6 +159,15 @@ export class Peer extends EventEmitter {
 				return;
 			}
 
+			// Protocol RCP request limiter LIP-0004
+			if (PROTOCOL_REMOTE_EVENT_RCP_EVENTS.has(rawRequest?.procedure)) {
+				if (this._protocolRCPCounter <= PROTOCOL_REMOTE_EVENT_RCP_EVENTS.size) {
+					this._protocolRCPCounter += 1;
```

Comment:
This implementation count each `REMOTE_EVENT_RPC_GET_PEERS_LIST` and `REMOTE_EVENT_RPC_GET_NODE_INFO` events together, although the LIP suggest limit only on `getPeers` separately. Can you explain it further why you combined both?

Reply:
If you are trying to guard only 1 `REMOTE_EVENT_RPC_GET_NODE_INFO` and `REMOTE_EVENT_RPC_GET_PEERS_LIST`, then this code will also work in case of two `REMOTE_EVENT_RPC_GET_PEERS_LIST` or two `REMOTE_EVENT_RPC_GET_NODE_INFO` requests.

Reply:
LIP-0004 mention both event as `should happen only at connection create`, with the given implementation we are allowing the expected scenario.